References: bsc#1131811

When the xenstored service is started it exits successfully but systemd seems to
lose track of the service and reports an error causing other xen services to fail.
This patch is a workaround giving systemd time to acknowledge a succesful start
of xenstored. The real fix is believed to be needed in systemd.

diff --git a/tools/hotplug/Linux/launch-xenstore.in b/tools/hotplug/Linux/launch-xenstore.in
index 991dec8d25..eb3d7c964c 100644
--- a/tools/hotplug/Linux/launch-xenstore.in
+++ b/tools/hotplug/Linux/launch-xenstore.in
@@ -79,6 +79,7 @@ test -f @CONFIG_DIR@/@CONFIG_LEAF_DIR@/xencommons && . @CONFIG_DIR@/@CONFIG_LEAF
 	echo -n Starting $XENSTORE_DOMAIN_KERNEL...
 	${LIBEXEC_BIN}/init-xenstore-domain $XENSTORE_DOMAIN_ARGS || exit 1
 	systemd-notify --ready 2>/dev/null
+	systemd-notify --booted 2>/dev/null && sleep 60
 
 	exit 0
 }
